.travis.yml

```diff
@@ -14,18 +28,14 @@ env:
 
 matrix:
   include:
-    - php: 5.6
-      env: DB=PGSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
```

Comment:
sorry, I'm not sure we can drop support for `4.4` in this minor branch of the module